platform/ios/src/MGLMapView.mm

```diff
@@ -3044,6 +3044,7 @@ - (void)removeStyleClass:(NSString *)styleClass
                 continue;
             }
             MGLAnnotationContext annotationContext = _annotationContextsByAnnotationTag.at(annotationTag);
+            NSAssert(_annotationContextsByAnnotationTag.count(annotationTag), @"Missing annotation for tag %u.", annotationTag);
```

Comment:
This assertion checks whether there’s an entry for `annotationTag` in `_annotationContextsByAnnotationTag`. There’s no way this assertion can fail, because we already have a guard a few lines above that `continue`s in that case. Instead, here, we need to assert that `annotationContext.annotation` is non-nil.